flow/project.py

```diff
@@ -2660,10 +2922,12 @@ def select(operation):
     def _generate_operations(self, cmd, jobs, requires=None):
         "Generate job-operations for a given 'direct' command."
         for job in jobs:
+            if len(job) > 1:
+                continue
             if requires and set(requires).difference(self.labels(job)):
                 continue
```

Comment:
combine

Reply:
This should not be combined as a label doesn't accept multiple jobs.
I'll use `elif` instead

Reply:
Python will short circuit if the logical operators if possible so `if len(job) > 1 or ...` if `len(job) > 1 == True` then it won't even run the right side of the or.

Reply:
Using an `elif` is fine, but just so you know.

Reply:
Oh okay, I didn't know about that. Thank you for letting me know.
If that's the case then I'll implement your suggestion